pkg/controller/controller.go

```diff
@@ -88,6 +93,10 @@ func NewUnmanaged(name string, mgr manager.Manager, options Options) (Controller
 		return nil, fmt.Errorf("must specify Name for Controller")
 	}
 
+	if options.Log == nil {
+		options.Log = log.NullLogger{}
```

Comment:
It feels a bit odd to be defaulting to `log.NullLogger` here rather than `logf.Log` as in other places. That said, ideally I think direct inheritance from the manager through the builder for the happy path would be a better happy path for the main use cases and use of the null logger when a logger is not provided for other use cases would be better than the use of a package level variable if we can.

Reply:
Yeah that makes sense, we can default to logf.Log here as well, in case folks want to disable the logging the value can be set to NullLogger

Reply:
Updated, ptal